lib/paperclip/storage/s3.rb

```diff
@@ -132,6 +132,7 @@ def self.extended(base)
         base.instance_eval do
           @s3_options     = @options[:s3_options] || {}
           @s3_permissions = set_permissions(@options[:s3_permissions])
+          @s3_acl_enabled = @options[:s3_acl_enabled] || true
```

Comment:
Thank you @everpcpc for the PR. This will not work if you user tries to set it to `false`. You perhaps need to do a `nil?` check.

Also, can you please also add tests for the same.

Reply:
Ah, yes you're right since this is a boolean. I've updated check and added tests.